lib/rules/no-html-safe.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const ERROR_MESSAGE = 'Do not use htmlSafe.';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'disallow the use of htmlSafe',
```

Comment:
Can you put `htmlSafe` in backticks to make it clear it's a function name?

Reply:
Looks like this isn't done yet. You will need to run `yarn update` after updating the message.

Reply:
Fix in: #1080